backends/edif/edif.cc

```diff
@@ -283,7 +337,9 @@ struct EdifBackend : public Backend {
 					RTLIL::SigSpec sig = sigmap(RTLIL::SigSpec(wire));
 					net_join_db[sig].insert(stringf("(portRef %s)", EDIF_REF(wire->name)));
 				} else {
-					*f << stringf("          (port (array %s %d) (direction %s))\n", EDIF_DEF(wire->name), wire->width, dir);
+					int bd[2]={0,wire->width-1};
+					int b[2]={bd[bounds[0]],bd[bounds[1]]};
```

Comment:
Check if `m = design->module(cell_it.second->type)` and `w = m->wire(port_it.first)` exist. Then use `GetSize(w)`, `w->upto` and `w->start_offset` to calculate `range_left` and `range_right` for `edif_names()`.